mage_ai/frontend/components/PipelineDetail/index.tsx

```diff
@@ -490,7 +541,21 @@ function PipelineDetail({
           const selectedBlockIndex =
             blocks.findIndex(({ uuid }: BlockType) => selectedBlock.uuid === uuid);
 
-          if (keyMapping[KEY_CODE_ESCAPE]) {
+          if (onlyKeysPresent([KEY_CODE_A], keyMapping) && !event.repeat && selectedBlockIndex !== -1) {
+            if (!isInputElement(event)) {
```

Comment:
In most cases, we don't need this because we already check that `textareaFocused = false` but this is here in case there's another input field (e.g. filter) on the page